Obsidian.Tests/Deserialization.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Obsidian.Net;
+using Obsidian.Net.Packets.Play;
+using Obsidian.Serializer;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Obsidian.Tests
+{
+    public class Deserialization
+    {
+        [Fact]
+        public void Handshake()
```

Comment:
It's not a good idea to include failing tests at the moment, the missing tests should be TODO comments, so we can take care about implementing them later

Reply:
I commented out all tests that weren't implemented.